### DISCUSSION TITLE
wincredman error over SSH via VSCode remote with GitHub CLI

### DISCUSSION THREAD

**Discussion by davetapley** [git-ecosystem/git-credential-manager#1346] [General]
Here's my workflow:

1. Connect to Windows machine using [VS Code Remote Development](https://code.visualstudio.com/docs/remote/remote-overview)
2. Use the [GitHub CLI](https://cli.github.com/)'s `gh repo clone` command

`push`/`pull` work, but every time I'm told (x3):
```
fatal: Unable to persist credentials with the 'wincredman' credential store.
See https://aka.ms/gcm/credstores for more information.
```

That link helpfully says:
https://github.com/git-ecosystem/git-credential-manager/blob/2fcbd770f6a3ef976a85c2463649aa0fb863c981/docs/credstores.md?plain=1#L36

---

But here's the thing: it does _seem_ to work, because I can `push`/`pull`.

What's going on here?
Can I suppress the `fatal` error?
Should that even be an error since it _appears_ to work regardless? 🤔

**Comment by junalmeida** [comment]
I face the same issue which is a huge blocker for me. It doesn't work, it asks for username and password then fails on git pull/push commands. `gh repo clone` works though. 
not sure how to workaround the issue

**Comment by hickford** [comment]
Have you tried [dpapi](https://github.com/git-ecosystem/git-credential-manager/blob/2fcbd770f6a3ef976a85c2463649aa0fb863c981/docs/credstores.md#dpapi-protected-files) the other credential store that's supported on Windows?

    git config --global credential.credentialStore dpapi

That might work over ssh, I'm not sure.
